ppr-ui/src/store/state/state-model.ts

```diff
@@ -86,6 +86,7 @@ export const stateModel: StateModelIF = {
       debtors: []
     }
   },
+  mhrRegistration: {},
```

Comment:
Model will be finalized later?

Reply:
Absolutely, as we build out the steps ( ie Describe your Home ), we will then build the model and Interface.

Reply:
Sounds good!